File: Auto-generated-tasks/Difficulty-2/Task-2/Solution.cs
int Kol = 8;
int[] A = new int[Kol];
int q = 0, n = Kol-1;
for (q = 0, q < Kol; q++) {
	A[q] = 8 + 880 * q;
}
for (n = Kol - 1; n > 1; n--) {
	Random rnd = new Random();
    int k = rnd.Next(n);
    int temp = A[n];
    A[n] = A[k];
    A[k] = temp;
}
for (q = 0; q < Kol; q++) {
	Console.WriteLine($"A[{q}]={A[q]}");
}

File: Auto-generated-tasks/Difficulty-2/Task-2/Solution.pas
Var
	Kol:=8, q:=0, n:=Kol-1, k:=0, temp:=0:integer;
	A:Array[0..Kol] of integer;
Begin
	for q:= 0 to Kol do begin
		A[q] := 8 + 880 * q;
	end;
	randomize;
	for n:= Kol - 1 downto 0 do begin
		k := random(n);
		
	end;
	for q:= 0 to Kol do begin
		WriteLn("A[", q, "]=", A[q]);
	end;
End.